### PR TITLE
Add note about EntityMoveEvent not firing for players

### PR DESCRIPTION
**Pull request by HexedHero**

### CODE REVIEW

**Review by NoahvdAa — APPROVED** [review]

**Review by MiniDigger — APPROVED** [review]

File: patches/api/0270-EntityMoveEvent.patch [hunk 1]
```diff
@@ -18,10 +18,13 @@ index 0000000000000000000000000000000000000000..fae0b1daf733e85d8f691f5ef934b8c0
 +import org.bukkit.event.Cancellable;
 +import org.bukkit.event.HandlerList;
 +import org.bukkit.event.entity.EntityEvent;
++import org.bukkit.event.player.PlayerMoveEvent;
```
Comment (qixils) [inline patches/api/0270-EntityMoveEvent.patch:21]: you can use fully qualified imports for javadocs (though if you choose not to then this should have a `// Paper`)

File: patches/api/0270-EntityMoveEvent.patch [hunk 2]
```diff
@@ -18,10 +18,13 @@ index 0000000000000000000000000000000000000000..fae0b1daf733e85d8f691f5ef934b8c0
...
 +import org.jetbrains.annotations.NotNull;
 +
 +/**
 + * Holds information for living entity movement events
++ * <p>
++ * Does not fire for players, please use {@link PlayerMoveEvent} instead.
```
Comment (qixils) [inline patches/api/0270-EntityMoveEvent.patch]: this *could* have a `<!-- Paper start -->` and end, though i think some of paper's javadoc patches don't add Paper comments so 🤷🏻‍♀️

**Review by qixils — APPROVED** [review]

File: patches/api/0270-EntityMoveEvent.patch (hunk 1 above)
Comment (NoahvdAa) [reply patches/api/0270-EntityMoveEvent.patch:21]: This is a paper event class though, so no // Paper is needed

File: patches/api/0270-EntityMoveEvent.patch (hunk 2 above)
Comment (NoahvdAa) [reply patches/api/0270-EntityMoveEvent.patch]: Probably also not necessary since this is a paper event class.

Comment (Proximyst) [reply patches/api/0270-EntityMoveEvent.patch]: In general, we don't care all that much about javadocs. They're not the most important thing in the world, and they only _really_ affect Paper's site.

File: patches/api/0270-EntityMoveEvent.patch (hunk 1 above)
Comment (qixils) [reply patches/api/0270-EntityMoveEvent.patch:21]: OH lol sorry 😅

**Pull request merged**